scipy/stats/_distn_infrastructure.py

```diff
@@ -1532,6 +1532,15 @@ def interval(self, confidence=None, *args, **kwds):
             end-points of range that contain ``100 * alpha %`` of the rv's
             possible values.
 
+        Notes
+        -----
+        This is typically implemented as ``ppf([p_tail, 1-p_tail])``, where
```

Comment:
drop "typically"
it is implemented

Reply:
Good idea. I meant that a distribution could override the generic implementation, but I don't think any do.